### DISCUSSION TITLE
Are there any plans to use the same backdrop material property as UWP in the future?

### DISCUSSION THREAD

**Discussion by otomad** [microsoft/WindowsAppSDK#3132] [Q&A]
The current way to set the backdrop material in WinUI 3 is to use the ISystemBackdropController interface (MicaController or DesktopAcrylicController), or use other nuget package likes WinUIEx. It seems too troublesome.

Are there any plans to use the same settings as UWP in the future?
```xaml
<Page muxc:BackdropMaterial.ApplyToRootOrPageBackground="True">
```

**Comment by christosk92** [comment]
I'm sure this can be replicated with a custom attached property and some code that can be found in WinUIEx(https://github.com/dotMorten/WinUIEx).

But yeh it would be nice to have this built-in! Although, you want to rename it because it behaves differently, it applies to a Window so maybe ``muxc:BackdropMaterial.<Mica/Acrylic>ApplyToWindow="True"``

**Reply by otomad** [reply]
That's true. After all, WinUIEx is even used in Microsoft's PowerToys. So I even suspect that WinUI 3 will give up using this prop.
